front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -139,19 +153,67 @@ export interface QueryParametersUpdate {
 export interface Props {
   ideaQueryParameters: QueryParameters;
   onUpdateQuery: (newParams: QueryParametersUpdate) => void;
+  showViewToggle?: boolean;
+  defaultView?: PresentationMode;
+  projectId?: string;
+  phaseId?: string;
+  title?: JSX.Element;
 }
 
-const IdeaCards = ({ ideaQueryParameters, onUpdateQuery }: Props) => {
+const IdeaCards = ({
```

Comment:
The new code in this file is really just copying over some parts from the IdeasWithoutFiltersSidebar so this component can be used on the Project page. There's definitely some code cleanup we can do on this existing code, but I thought we could add it to the code cleanup ticket instead for now? :thinking:

Reply:
I think your approach is fine. Let's put cleaning up this one on top of the list then.